.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,4 +9,9 @@
 
 * [ ] This PR adds new unconventional dependencies following the process described [here](../CONTRIBUTING.md/#adding-new-unconventional-dependencies)
 
+### New Features
+
+* [ ] Have you added the new feature to an [integration test](../INTEGRATION_TESTS.md)?
```

Comment:
I'm not convinced this link is going to work when viewed in a pull request...?

Reply:
yeah you are right. Updated all the links in the template.